http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -456,6 +531,39 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
 
     builderWithAutoscaling.build()
   }
+
+  private[util] def buildGalaxyChartOverrideValuesString(appName: AppName,
+                                                         cluster: KubernetesCluster,
+                                                         nodepoolName: NodepoolName,
+                                                         userEmail: WorkbenchEmail): String = {
+    val release = config.galaxyAppConfig.releaseName
+    val proxyUrl = host(cluster, config.proxyConfig.proxyDomain).toString
+    val hostUrl = config.proxyConfig.getProxyServerHostName
+    // TODO Is the path below templated in any config? If not, consider adding
+    val ingressPath = s"/proxy/google/v1/apps/${cluster.googleProject.value}/${appName.value}/galaxy"
+
+    // Using the string interpolator raw""" since the chart keys include quotes to escape Helm
+    // value override special characters such as '.'
+    // https://helm.sh/docs/intro/using_helm/#the-format-and-limitations-of---set
```

Comment:
^^ may no longer be necessary since not using quotes in keys work the way we want.